presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/WindowMatcher.java

```diff
@@ -46,24 +46,23 @@ public boolean matches(PlanNode node, Session session, Metadata metadata, Symbol
         }
 
         WindowNode windowNode = (WindowNode) node;
-        Collection<FunctionCall> actualCalls = windowNode.getWindowFunctions().values();
+        List<FunctionCall> actualCalls = windowNode.getWindowFunctions().values().stream().map(function -> function.getFunctionCall()).collect(toList());
```

Comment:
break this onto multiple lines like:
```java
blah.stream()
.map(...)
.collect(...)
```
Also I think "function -> function.getFunctionCall()" can be simplified as Function::getFunctionCall?